reactioncommerce/reaction
# feat: 4868 - tag management operator UI

Resolves #4868   
Impact: **major**  
Type: **feature**

## Issue

Add a tag management UI

## Solution

Add a tag management UI that can

1. Show a list of all tags
2. Create / edit tags
3. Edit metadata for tags
4. Upload / select hero images
5. Re-order products assigned to a tag using a priority

## Breaking changes

none

## Testing
1. open http://localhost:3000/operator/tags
2. Add / edit / delete tags
3. Edit positions of products and see that the values save

### Starter Kit
1. Test that metafields show up and can be edited/displayed
2. Test that isVisible/isHidden work and is displayed on Starter Kit
3. Test that product re-ordering works and is displayed on Starter Kit
4. Test that changing DisplayTitle works and is displayed on Starter Kit
5. Test that changing HeroImage works and is displayed on Starter Kit
